daemon/api.go

```diff
@@ -118,6 +121,17 @@ var (
 		GET:    getOpInfo,
 		DELETE: deleteOp,
 	}
+
+	capabilitiesCmd = &Command{
+		Path: "/1.0/capabilities",
+		GET:  getCapabilities,
+		POST: createCapability,
```

Comment:
The "creation" of a capability sounds like language that should be used for types. For adding capabilities to the repository we've used "Add", so it sounds reasonable to reused the same term.